app/Jobs/NotifyUserOfNewGistComment.php

```diff
@@ -20,9 +20,9 @@ class NotifyUserOfNewGistComment extends Job implements ShouldQueue
 
     public function __construct($user, $comment, $gist)
     {
-        $this->user = $user;
+        $this->user    = $user;
```

Comment:
We don't align `=` :)

I promise, we're working on a Tighten code style that will be available soon!

Reply:
Already fixed this 😺